src/parser/ad_parser.js

```diff
@@ -79,6 +79,7 @@ function parseInLine(adElement, emit, { allowMultipleAds } = {}) {
  * @return {Object} ad - The ad object.
  */
 function parseAdElement(adTypeElement, emit) {
+  let adVerificationFromExtensions = [];
```

Comment:
It should be in plural 
```suggestion
  let adVerificationsFromExtensions = [];
```